en/peripherals/mavlink_peripherals.md

```diff
@@ -1,24 +1,46 @@
-the optimal mode depending on the application scenario.
-
--   Companion computer mode at 921600 baud
--   Companion computer mode at 57600 baud
--   OSD mode at 57600 baud
--   Command / RC input mode (receive only) at 57600 baud
--   Normal telemetry mode at 57600 baud
-
-> **Note** OSD mode defines the MAVLink stream required by OSD (set of activated messages & their rates). The other modes are for other specific hardware (e.g. ESP8266 for WiFi), and use the optimal baud rate settings for their link.
-
-![QGC Telemetry Setup](../../images/qgc_telemetry_setup.png)
-
+# MAVLink Peripherals (GCS/OSD/Companion)
+
+Ground Control Stations (GCS), On-Screen Displays (OSD), Companion Computers, and other MAVLink peripherals interact with PX4 using separate MAVLink streams, sent via different serial ports.
+These communication channels are are configured using the [MAVLink parameters](../advanced_config/parameter_reference.md#mavlink).
+
+## MAVLink Instances
+
+In order to assign a particular peripheral to a serial port we use the (abstract) concept of a *MAVLink instance*. 
+Each instance can represent a particular set of streamed messages (see "mode" below); parameters are used to define the set of messages, the port used, data rate, etc.
+
+> **Note** At time of writing three instances are defined, which correspond to the 0, 1, 2 in the parameters listed below.
+
+The parameters for each instance are:
+- [MAV_X_CONFIG](../advanced_config/parameter_reference.md#MAV_0_CONFIG) - Set the serial port (UART) for this instance "X", where X is 0, 1, 2. 
+  It can be any unused port, e.g.: TELEM2, TELEM3, GPS2 etc. For more information see [Serial Port Configuration](../peripherals/serial_configuration.md).
+- [MAV_X_MODE](../advanced_config/parameter_reference.md#MAV_0_MODE) - Specify the telemetry mode/target (the set of messages to stream for the current instance and their rate).
+  The values are:
+  - *Normal*: Standard set of messages for a GCS.
+  - *Custom*: Nothing (in the default PX4 implementation).
+  - *Onboard*: Standard set of messages for a companion computer.
+  - *OSD*: Standard set of messages for an OSD system.
+  - *Magic*: Nothing (in the default PX4 implementation).
+  - *Config*: Set of messages that may be useful for configuration.
+  - *Iridium*: Standard set of messages for an Iridium satellite system. This should not be set (use [ISBD_CONFIG](../advanced_features/satcom_roadblock.md) to configure a satellite system).
+  - *Minimal*: Minimal set of messages.
+- [MAV_X_RATE](../advanced_config/parameter_reference.md#MAV_0_MODE) - Set the maximum data rate for this instance (bytes/second). 
+  This is the combined rate for all streams of individual message (the rates for individual messages are reduced if the total rate exceeds this value).
+- [MAV_X_FORWARD](../advanced_config/parameter_reference.md#MAV_0_FORWARD) - Enable forwarding of MAVLink packets received by the current instance onto other interfaces.

```

Comment:
You would enable it for example if you have a mavlink-enabled camera connected, so that a GCS (connected to another port on the pixhawk) can directly talk to the camera.